app/src/main/java/com/veriff/demo/data/dataSources/login/UserDataSourceI.kt

```diff
@@ -0,0 +1,12 @@
+package com.veriff.demo.data.dataSources.login
+
+import com.veriff.demo.data.dataSources.DataSourceCallback
+
+interface UserDataSourceI {
```

Comment:
Not a fan of using `...I ` for interfaces, I'd rather add `Default...` or `...Impl` to implementation, because the implementation name is usually hidden into DI

Reply:
Yea that makes more sense 👍🏽